server/server_test.go

```diff
@@ -294,6 +295,33 @@ func TestMiddlewareLogging(t *testing.T) {
 	http.DefaultClient.Do(req)
 }
 
+func TestMuxMiddleware(t *testing.T) {
+	var level logging.Level
+	level.Set("info")
+	cfg := Config{
+		HTTPListenAddress: "localhost",
+		HTTPListenPort:    9193,
```

Comment:
why not put this one on a unique port number rather than moving the ones later on?

or generate unique numbers using `iota`

Reply:
Using a unique port number now.